src/playwright-runner.ts

```diff
@@ -332,6 +340,10 @@ async function runPlaywright(
     PLAYWRIGHT_JUNIT_OUTPUT_NAME: runCfg.junitFile,
     SAUCE_REPORT_OUTPUT_NAME: runCfg.sauceReportFile,
     FORCE_COLOR: '0',
+    SAUCE_WEB_ASSETS_DIR:
+      suite.env?.SAUCE_SYNC_WEB_ASSETS.toLowerCase() === 'true'
+        ? runCfg.assetsDir
+        : '',
```

Comment:
Same as the comment above (relocate logic), and then you could do:
```suggestion
    SAUCE_WEB_ASSETS_DIR: runCfg.webAssetsDir,
```